### PR TITLE
Add prioritisation option for sequence process

### PR DESCRIPTION
**Pull request by timothyylim**
Just a staging PR. Not ready for merging just yet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "webpack-cli": "^5.1.1"
   },
   "dependencies": {
+    "@babel/core": "^7.23.9",
```
Comment (otacke) [inline package.json:39]: Good idea to update dependencies as well. Running something like https://www.npmjs.com/package/npm-check-updates manually once done and committing module updates in a separate commit would be totally fine. But handling this will be part of repository maintenance any way.

File: semantics.json [hunk 2]
```diff
@@ -113,6 +113,14 @@
     "importance": "low",
     "description": "These options will let you control the behaviour.",
     "fields": [
+      {
```
Comment (otacke) [inline semantics.json:116]: When you change the structure of `semantics.json` or strings, please remember to also change the translation files accordingly. The H5P CLI tool has a utility command to `update-translations`. You can add a property `state` to a field in `semantics.json` with a value of `new`, `updated` or `removed` and the H5P CLI will change the structure/update labels accordingly. Moving fields around will involve manual changes to the translation files, however.

So, here you could have added `"state": "new"`, and running the `update-translations` command will add the respective fields to the translation files.

File: semantics.json [hunk 3]
```diff
@@ -113,6 +113,14 @@
...
+        "name": "prioritizeable",
+        "type": "boolean",
+        "label": "Prioritization",
+        "importance": "high",
+        "default": false,
+        "description": "Tick this off if you want the learner to prioritize the statements"
```
Comment (otacke) [inline semantics.json:122]: It's convention that the values of `description` fields always end with a `.`.

**Review by otacke — CHANGES_REQUESTED** [review]
Looks good in general! It's essentially just updating the translation files according to the changes in `semantics.json` that seems to be missing.

**Comment by otacke** [comment]
@timothyylim Seems this is still to be done ;-)

File: package.json (hunk 1 above)
Comment (timothyylim) [reply package.json:39]: Alright, shall we just keep this as part of the PR then?

File: semantics.json (hunk 2 above)
Comment (timothyylim) [reply semantics.json:116]: Great tip. Thanks!

Comment (timothyylim) [reply semantics.json:116]: Language files have now been updated.

File: package.json (hunk 1 above)
Comment (otacke) [reply package.json:39]: This can be left out, yes.
